src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -94,11 +96,23 @@
 			:loan-amount="loan.loanAmount"
 		/>
 
-		<!-- Button -->
+		<!-- CTA Button -->
 		<kv-loading-placeholder
 			v-if="isLoading"
 			class="tw-rounded" :style="{width: '9rem', height: '3rem'}"
 		/>
+
+		<kv-button
+			v-if="!isLoading"
+			class="tw-mb-2"
+			:to="`/lend/${loan.loanId}`"
```

Comment:
It shouldn't need it since it seems like it's the default, but if you add variant="primary" does it work correctly?